docs/contribution.html

```diff
@@ -16,52 +18,71 @@
-        <p>Additional information can be found <a href="https://github.com/MrTroble/TGEngine/blob/master/CONTRIBUTING.md">here</a></p>
-        <h1 id="standarts">Standarts</h1>
-        <p>Every pull request needs to meet certain standarts. And thus are as following.</p>
-        <p><em>Note: Currently most standarts are not used yet by the engine itself. This is going to be changed asap</em></p>
+        <p><em><strong>Please review the <a
+                       href="https://github.com/MrTroble/TGEngine/blob/master/CODE_OF_CONDUCT.md">Code of Conduct</a>
+                    before contributing in any way to the repository!</strong></em></p>
+        <p>Additional information can be found <a
+               href="https://github.com/MrTroble/TGEngine/blob/master/CONTRIBUTING.md">here</a></p>
+
+        <h1 id="standards">Standards</h1>
+        <p>Every pull request needs to meet certain standards.</p>
+        <p><em>Note: Most standards are not used yet by the engine itself at the current date. This is going to be
+                changed as soon as possible.</em></p>
         <div id="subcontainer">
             <div class="submenu">
-                <a href="#includes">Includes / Header</a><br>
-                <a href="#variables">Variables / Makros</a><br>
-                <a href="#methodes">Methodes</a><br>
+                <a href="#general">General</a>
+                <a href="#includes">Includes, Headers</a><br>
+                <a href="#variables">Variables, Macros</a><br>
+                <a href="#methods">Methods</a><br>
                 <a href="#modules">Modules</a><br>
-                <a href="#classes">Class / Struct / Union</a>
+                <a href="#classes">Classes, Structs, Unions</a>
             </div>
         </div>
-        <h2 id="includes">Includes / Header</h2>
+        <h2 id="general">
+        <ul>
+            <li>Everything should be put in namespaces</li>
+            <li>Do not use "set namespace", use tge::namespace instead</li>
+            <li>Do not use anything but vectors for dynamic lists</li>
+        </ul>
+        </h2>
+        <h2 id="includes">Includes, Headers</h2>
         <ul>
             <li>
                 The first statement in a header file should be<br>
                 <pre class="prettyprint">#pragma once</pre>
-                <em><p>Note: due to the pragma preprocessor not being a standart this might not be possible for your compiler</p></em>
+                <em>
+                    <p>Note: This might not be possible for your compiler due to the pragma preprocessor not being a
+                        standard</p>
+                </em>
             </li>
             <li>The second statements are the includes</li>
-            <li>Each .cpp source file should include it's according header file and nothing else</li>
+            <li>Each .cpp source file should include its according header file and nothing else</li>
```

Comment:
This line needs to be removed for changed, as it has been used very often frequently to reduce compile time and remove circular dependencies.